client/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "ion-extension-client",
-	"version": "0.2.0",
+	"version": "0.0.1",
```

Comment:
See https://stackoverflow.com/questions/46653833/is-there-a-way-to-force-npm-to-generate-package-lock-json for how to force an update on package-lock.json

Reply:
Sure. I will use `npm install --package-lock` to force the update for package-lock.json file.